CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 1.3.1 (unreleased)
 ==================
 
+Other
+-----
+
+- Add units to BARTDELT and HELIDELT keywords in datamodels schmea. [#147]
```

Comment:
typo in "schmea" and should probably also mention that these are *jwst* keywords (now that it's part of a generic package).